documentation/versioned_docs/version-1.x/examples/cad-explode.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: cad-explode
+title: Expand asset tool
+description: Demo use the expand tool to expond and inspect a given asset.
+---
+
+import { DemoWrapper } from '@site/src/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section describes the expand asset tool, which contains helper functions and logic for inspecting and expanding a given asset.
+The constructor of this tool needs to be supplied with a [tree index](../concepts.md) which is the tree index of the root of the asset, and an instance of the `Cognite3DModel` that the asset belongs to.
+
+:::note
+For performance reasons, the root tree index must have a subtree size of less than 1000, else it will throw an error.
+:::
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+`ExpandAssetTool` contains two instance methods: `expand` and `reset`.
+`expand` takes a float describing how far the individual nodes should be expanded (this will very based on the initial vector between the node and root).
```

Comment:
```suggestion
`expand` takes a float describing how far the individual nodes should be expanded (this will vary based on the initial vector between the node and root).
```